client/src/App.js

```diff
@@ -11,6 +11,7 @@ import './stylesheets/queerInput.css';
 import './stylesheets/queerSnackbar.css';
 import './stylesheets/tooltips.css';
 import './stylesheets/resultCard.css';
+import './stylesheets/defineForm.css';
```

Comment:
what do you think of importing this in DefineForm.js?

Reply:
Makes sense to me. Only included it in `App.js` out of deference to the existing `import` statements.